libvast/test/segment_store.cpp

```diff
@@ -29,9 +29,9 @@ FIXTURE_SCOPE(segment_store_tests,
               fixtures::deterministic_actor_system_and_events)
 
 TEST(construction and querying) {
-  // FIXME: use directory from fixture
-  rm("foo");
-  auto store = segment_store::make(sys, path{"foo"}, 512_KiB, 2);
+  auto path = directory / "segments";
+  rm(path);
```

Comment:
The fixture always invokes `rm(directory)` in its constructor so you don't have to.